sns/parity_ops.c

```diff
@@ -27,6 +27,7 @@
 #include "lib/assert.h"
 #include "sns/parity_ops.h"
 
+#ifdef __KERNEL__
```

Comment:
I think we should keep the m0_parity_fini() and m0_parity_init() always defined for user space and kernel space.
```
M0_INTERNAL void m0_parity_fini(void)
{
#ifdef __KERNEL__
        galois_calc_tables_release();
#endif
}

M0_INTERNAL int m0_parity_init(void)
{
#ifdef __KERNEL__
        int ret = galois_create_mult_tables(M0_PARITY_GALOIS_W);
        M0_ASSERT(ret == 0);
#endif 
        return 0;
}
```

Reply:
Done.